docs/content/querying/lookups.md

```diff
@@ -0,0 +1,191 @@
+# Lookups
+
+Lookups are a concept in Druid where dimension values are (optionally) replaced with a new value.
+
+Lookups can be specified as part of the runtime properties file.
+ ```json
+ druid.query.extraction.namespaceList=\
+   [{ "type":"uri", "namespace":"some_uri_lookup","uri": "file:/tmp/prefix/",\
+   "namespaceParseSpec":\
+     {"type":"csv","columns":["key","value"]},\
+   "pollPeriod":"PT5M"},\
+   { "type":"jdbc", "namespace":"some_jdbc_lookup",\
+   "connectorConfig":{"createTables":true,"connectURI":"jdbc:mysql://localhost:3306/druid","user":"druid","password":"diurd"},\
+   "table": "lookupTable", "keyColumn": "mykeyColumn", "valueColumn": "MyValueColumn", "tsColumn": "timeColumn"}]
+ ```
+
+## URI lookup update
+The remapping values for each lookup can be specified by a lookup update json object as per
+```json
+{
+  "type":"uri",
+  "lookup":"some_lookup",
+  "uri": "s3://bucket/some/key/prefix/",
+  "lookupParseSpec":{
+    "type":"csv",
+    "columns":["key","value"]
+  },
+  "pollPeriod":"PT5M",
+  "versionRegex": "renames-[0-9]*\\.gz"
+}
+```
+
+The `pollPeriod` value specifies the period in ISO 8601 format between checks for updates. If the source of the lookup is capable of providing a timestamp, the lookup will only be updated if it has changed since the prior tick of `pollPeriod`. A value of 0, an absent parameter, or `null` all mean populate once and do not attempt to update. Whenever an update occurs, the updating system will look for a file with the most recent timestamp and assume that one with the most recent data.
+
+The `versionRegex` value specifies a regex to use to determine if a filename in the parent path of the uri should be considered when trying to find the latest version. Omitting this setting or setting it equal to `null` will match to all files it can find (equivalent to using `".*"`). The search occurs in the most significant "directory" of the uri.
+
+The lookupParseSpec can be one of a number of values. Each of the examples below would rename foo to bar, baz to bat, and buck to truck. All parseSpec types assumes each input is delimited by a new line.
```

Comment:
out of curiosity, why do we need so many parse specs? can the lookup just have a standard format?